src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -10,4 +10,83 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.gson.Gson;
+import com.google.sps.data.Comment;
+import java.io.IOException;
+import java.lang.Integer;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/data")
+public class DataServlet extends HttpServlet {
+
+  private static final SimpleDateFormat dateFormat = new SimpleDateFormat("MM/dd/yyyy");
+  
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query(Comment.MESSAGE_KEY).addSort(Comment.TIMESTAMP_KEY, SortDirection.DESCENDING);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    ArrayList<Comment> comments = new ArrayList<Comment>(); 
+
+    for(Entity entity : results.asIterable(FetchOptions.Builder.withLimit(10))) {
```

Comment:
is this on purpose? So basically this will only fetch the first 10 comments right? You may want to add comments indicating that this is for MVP for now.

Nit: I would extract `results.asIterable(FetchOptions.Builder.withLimit(10))` into a variable like 
```
PreparedQuery preparedQuery = datastore.prepare(query);
// For MVP, only fetch the first 10 comments for now.
Iterable<Entity> entities = preparedQuery.asIterable(FetchOptions.Builder.withLimit(10));
for (Entity entiry : entities) {
``` 

to make the code more clear.